### DISCUSSION TITLE
code diffs look weird

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3683] [Mobile] [answered]
diff taken from https://github.com/flutter/flutter/issues/79371

i would expect code diffs to looks the same on all platforms

1. ios dark mode - diffs have light background
2. ios light mode - ok, identical with web
3. android light and dark mode - totally broken and different from web

<details>
<summary>screenshots</summary>

| | web | ios | android |
| - | -| - | - |
| dark | ![web](https://user-images.githubusercontent.com/39104740/113487248-b5852b80-94bf-11eb-82a1-969601f36568.png) | ![ios](https://user-images.githubusercontent.com/39104740/113487253-bb7b0c80-94bf-11eb-89eb-37e8420ff041.PNG) | ![android](https://user-images.githubusercontent.com/39104740/113487258-c5047480-94bf-11eb-9d35-3c92933cb705.jpg) |
| light | ![web_light](https://user-images.githubusercontent.com/39104740/113487262-cafa5580-94bf-11eb-9f27-8e0b4cf20f47.png) | ![ios_light](https://user-images.githubusercontent.com/39104740/113487270-d3529080-94bf-11eb-9072-c074c202db40.PNG) | ![android_light](https://user-images.githubusercontent.com/39104740/113487273-d51c5400-94bf-11eb-8790-d7aab3c907c2.jpg) |
</details>

**Comment by brianlovin** [comment, marked as answer]
Thank you! I'm filing issues for both the iOS and Android bugs now.
